### PR TITLE
DAT-18897: added databricks specific configurations to the XSD element 'extendedTableProperties'

### PR DESCRIPTION
**Pull request by SvampX**

### CODE REVIEW

File: src/main/java/liquibase/ext/databricks/change/createTable/CreateTableChangeDatabricks.java [hunk 1]
```diff
@@ -71,13 +91,4 @@ protected CreateTableStatement generateCreateTableStatement() {
     public ExtendedTableProperties getExtendedTableProperties() {
         return extendedTableProperties;
     }
-
-    @Override
-    public String getSerializableFieldNamespace(String field) {
-        if("clusterColumns".equalsIgnoreCase(field)) {
```
Comment (filipelautert) [inline src/main/java/liquibase/ext/databricks/change/createTable/CreateTableChangeDatabricks.java:77]: don't we need to add a 
```
    @Override
    public String getSerializedObjectNamespace() {
        return "http://www.liquibase.org/xml/ns/databricks";
    }
```

here?

File: src/main/java/liquibase/ext/databricks/change/createTable/CreateTableChangeDatabricks.java [hunk 2]
```diff
@@ -15,6 +14,11 @@
 @DatabaseChange(name = "createTable", description = "Create Table", priority =  PrioritizedService.PRIORITY_DATABASE)
 @Setter
 public class CreateTableChangeDatabricks extends CreateTableChange {
+    private static final String DOUBLE_INIT_ERROR = "Double initialization of extended table properties is not allowed. " +
+            "Please avoid using both EXT createTable attributes and Databricks specific extendedTableProperties element. " +
+            "Element databricks:extendedTableProperties is preferred way to set databricks specific configurations.";
+    private static final String PARTITION_CLUSTER_COLLISION_ERROR = "Databricks does not support CLUSTER columns " +
+            "AND PARTITION BY columns, please pick one. And do not supply the other";
```
Comment (filipelautert) [inline src/main/java/liquibase/ext/databricks/change/createTable/CreateTableChangeDatabricks.java:21]: I think the` . And do not supply the other`  could be removed.

**Review by filipelautert — CHANGES_REQUESTED** [review]

File: src/main/java/liquibase/ext/databricks/change/createTable/CreateTableChangeDatabricks.java (hunk 1 above)
Comment (SvampX) [reply src/main/java/liquibase/ext/databricks/change/createTable/CreateTableChangeDatabricks.java:77]: We used to have an issue with marking `createTable` change type as `databricks:createTable` when tried to use ext schema for generate snapshot. 
Like following:
```
<changeSet author="msav1 (generated)" id="1729176154905-4">
        <databricks:createTable clusterColumns="test_id,test_present_new" tableName="test_table_clustered_new">
            <column name="test_id" type="INT"/>
            <column name="test_present_new" type="INT"/>
            <databricks:extendedTableProperties tblProperties="delta.checkpoint.writeStatsAsJson=false,delta.checkpoint.writeStatsAsStruct=true,delta.checkpointPolicy=v2,delta.columnMapping.maxColumnId=3,delta.columnMapping.mode=name,delta.enableDeletionVectors=true,delta.enableRowTracking=true,delta.feature.allowColumnDefaults=supported,delta.feature.clustering=supported,delta.feature.columnMapping=supported,delta.feature.deletionVectors=supported,delta.feature.domainMetadata=supported,delta.feature.rowTracking=supported,delta.feature.v2Checkpoint=supported,delta.minReaderVersion=3,delta.minWriterVersion=7,delta.rowTracking.materializedRowCommitVersionColumnName=_row-commit-version-col-d5786a86-a84a-4ab1-a96d-b08b90d5fadf,delta.rowTracking.materializedRowIdColumnName=_row-id-col-068f972c-24c9-43af-ae82-a76948dfd520"/>
        </databricks:createTable>
    </changeSet>
```
I've retested it now with suggested namespace and now we have following, notice `tableName` attribute:
```
    <changeSet author="msav1 (generated)" id="1730194147057-3">
        <createTable databricks:tableName="test_reference">
            <column name="test_id" type="BIGINT">
                <constraints nullable="false" primaryKey="true" primaryKeyName="test_reference_pk"/>
            </column>
            <column name="test_float" type="FLOAT">
                <constraints nullable="false"/>
            </column>
            <databricks:extendedTableProperties tblProperties="'delta.checkpoint.writeStatsAsJson'='false', 'delta.checkpoint.writeStatsAsStruct'='true', 'delta.enableDeletionVectors'='true', 'delta.feature.allowColumnDefaults'='supported', 'delta.feature.deletionVectors'='supported', 'delta.feature.identityColumns'='supported', 'delta.feature.invariants'='supported', 'delta.minReaderVersion'='3', 'delta.minWriterVersion'='7'"/>
        </createTable>
    </changeSet>
```

Comment (SvampX) [reply src/main/java/liquibase/ext/databricks/change/createTable/CreateTableChangeDatabricks.java:77]: Same generated changeset without overriden `getSerializedObjectNamespace ()`:
```
    <changeSet author="msav1 (generated)" id="1730195432421-3">
        <createTable tableName="test_reference">
            <column name="test_id" type="BIGINT">
                <constraints nullable="false" primaryKey="true" primaryKeyName="test_reference_pk"/>
            </column>
            <column name="test_float" type="FLOAT">
                <constraints nullable="false"/>
            </column>
            <databricks:extendedTableProperties tblProperties="'delta.checkpoint.writeStatsAsJson'='false', 'delta.checkpoint.writeStatsAsStruct'='true', 'delta.enableDeletionVectors'='true', 'delta.feature.allowColumnDefaults'='supported', 'delta.feature.deletionVectors'='supported', 'delta.feature.identityColumns'='supported', 'delta.feature.invariants'='supported', 'delta.minReaderVersion'='3', 'delta.minWriterVersion'='7'"/>
        </createTable>
    </changeSet>
```

**Review by KushnirykOleh — APPROVED** [review]

**Review by filipelautert — APPROVED** [review]

**Pull request merged**